### PR TITLE
Do not propage group config to autoprovision device

### PR DESCRIPTION
**Pull request by AlvaroVega**
issue https://github.com/telefonicaid/iotagent-node-lib/issues/1377

This way autorpovisioned devices with remain without some flags defined like timestamp, expressionLanguage (which will take in runtime from group configuration)

- [x] Similar PRs in iotagent-json and iotagent-ul to align with local  findOrCreate

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1 +1,2 @@
+- Do not propage group config (timestamp, expresisonLanguage) to autoprovision device (#1391)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- Fix: do not propagate group config (timestamp and explicitAttrs) to autoprovisioned devices (at database level) (#1391)
```

Note that expressionLanguage was removed in a recent version, so it doens't belong to the existing API or DB model.

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -1 +1,2 @@
+- Fix: do not propagate group config (timestamp and explicitAttrs) to autoprovisioned devices (at database level) (#1391)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- Fix: do not propagate group config (timestamp and explicitAttrs) to autoprovisioned devices (at database level) (#1377)
```

**Comment by mapedraza** [comment]
After comment https://github.com/telefonicaid/iotagent-ul/pull/616#issuecomment-1609357664

Added documentation about group parameters propagation here: https://github.com/telefonicaid/iotagent-node-lib/pull/1391/commits/64f4ba31544fc48de7ad11d7eb373ec8e9f81735

**Review by fgalan — APPROVED** [review]
LGTM

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**